mailpoet/lib/Config/Hooks.php

```diff
@@ -424,11 +424,15 @@ public function setupSettingsLinkInPluginPage() {
     );
   }
 
-  public function setSettingsLinkInPluginPage($actionLinks) {
+  /**
+   * @param array<string, string> $actionLinks
+   * @return array<string, string>
+   */
+  public function setSettingsLinkInPluginPage(array $actionLinks): array {
     $customLinks = [
-      'settings' => '<a href="' . $this->wp->adminUrl('admin.php?page=mailpoet-settings') . '" aria-label="' . esc_attr__( 'View MailPoet settings', 'mailpoet' ) . '">' . esc_html__( 'Settings', 'mailpoet' ) . '</a>',
+      'settings' => '<a href="' . $this->wp->adminUrl('admin.php?page=mailpoet-settings') . '" aria-label="' . $this->wp->escAttr(__('View MailPoet settings', 'mailpoet')) . '">' . $this->wp->escHtml(__('Settings', 'mailpoet')) . '</a>',
```

Comment:
@lysyjan I'm ok with using the internal functions and I already approved the PR, but just to make sure I'm not missing something, I thought that for translation functions we use WP functions directly?

Reply:
I used internal functions for escaping the attribute and HTML, but for translation is still used WP function directly.

Reply:
Thanks for the input. I guess I see `esc_attr__()` and `esc_html__()` as translation functions that also take care of escaping and that is why I used the WP functions instead of the ones that we define.